src/Controls/src/Core/Button/Button.Android.cs

```diff
@@ -20,5 +21,14 @@ public static void MapLineBreakMode(IButtonHandler handler, Button button)
 		{
 			handler.PlatformView?.UpdateLineBreakMode(button);
 		}
+
+		internal static void MapRippleColor(IButtonHandler handler, Button button)
+		{
+			if(button.IsSet(PlatformConfiguration.AndroidSpecific.Button.RippleColorProperty))
```

Comment:
What happens if the user does a clear value? If we set the color to red and then unset, it will stay red. It should maybe pass null and the handler can assume that null == default?